packages/block-library/src/post-navigation-link/index.php

```diff
@@ -31,6 +31,10 @@ function render_block_core_post_navigation_link( $attributes, $content ) {
 	$styles = '';
 	if ( isset( $attributes['style']['typography']['writingMode'] ) ) {
 		$styles = "writing-mode: {$attributes['style']['typography']['writingMode']};";
+
+		if ( isset( $attributes['style']['typography']['textOrientation'] ) ) {
+			$styles .= " text-orientation: {$attributes['style']['typography']['textOrientation']};";
+		}
```

Comment:
This should be removed, but it is a question of timing. The plan now is to update the safe CSS filter instead: 
https://github.com/WordPress/gutenberg/pull/54581